### ISSUE TITLE
Issue with directory views/blocks in v0.6

### ISSUE THREAD

**Issue by aidanweltner** [MWDelaney/sage-acf-wp-blocks#50]
I know that you just pushed the update for version 0.6 but I found a bug with PR #46. The default folder for blocks is now `views/block` instead of `views/blocks` as it is in all the documentation. I just got an error after updating to version 0.6, 

> Fatal error: Uncaught Exception: View [views.blocks.<block>] not found.

The error states that it is looking in the correct folder but only goes away when I change that folder name to `views/block`. I don't see exactly where that would be happening in that code but I figure it's worth bringing up since the docs reflect `views/blocks`.

**Comment by robmeijerink** [comment]
This shouldn't happen since the paths that are checked are still correct. Are you sure that you have a block.blade.php file in your views/blocks folder?

**Comment by speakerbug** [comment]
Just commenting to say that I received the same error as @aidanweltner after updating.

**Comment by aidanweltner** [comment]
@robmeijerink yes, I didn't change any files or any names and all my blocks were working with v0.5. The error only showed after I updated the package with Composer. 

I should also clarify that this project was with Sage 9.

**Comment by robmeijerink** [comment]
Thanks, I have found the issue.

**Comment by djmtype** [comment]
After updating to sage-acf-wp-blocks v0.6 I'm also having this problem. 

If I don't update my directory from `blocks` to `block`, Wordpress throws an error. 

If I do, then none of my blocks can be found.

**Comment by joshuafredrickson** [comment]
This is fixed in 0.7.
